principia_make.sh

```diff
@@ -1,9 +1,12 @@
-./autogen.sh
-./configure \
-    CC=clang \
-    CXX=clang++ \
-    CFLAGS="${C_FLAGS?}" \
-    CXXFLAGS="${CXX_FLAGS?}" \
-    LDFLAGS="${LD_FLAGS?}" \
-    LIBS="-lc++ -lc++abi"
+mkdir -p build
```

Comment:
Is this going to put the build artifacts in a different place (the `build` directory)?  If it does it's going to require changes in the Prinicipia `Makefile` and in the azure pipeline, which is annoying.  Can we use `cmake` and keep the artifacts at the same place?

Reply:
Pushed a commit that puts the important artifacts where they used to be. Isn't pretty, but it (hopefully) works.